hc-chart/hc-chart-table.html

```diff
@@ -98,15 +98,17 @@
     <g class="grey small" role="row"><!-- moved from end into table: x-axis labels -->
       <text x="334.5" text-anchor="middle" class="grey" y="306" role="rowheader">Time</text>
       <text x="130" text-anchor="middle" y="269" role="columnheader">January 2009</text>
-      <text x="232" text-anchor="middle" y="269" role="columnheader"><tspan>December</tspan><tspan dy="14" x="232">2010</tspan><title>December 2010</title></text>
```

Comment:
I'll take the PR and see what this gives visually - it came from the original chart, and there might be enough space to fit the text anyway. Otherwise I might look at reverting this change and finding another way to make it work…